kuksa_databroker/createbom/createbom.py

```diff
@@ -117,7 +117,7 @@ def generate_bom(source_path, target_path, dashout):
     errors = []
     for crate in crates:
         try:
-            print(f"License for {crate['name']}: ", end="")
+            print(f"License for {crate['name']} {crate['version']}: ", end="")
```

Comment:
This is to get output with version like:

`2023-07-31T16:02:41.7150426Z #0 6.859 License for matchit 0.7.1: MIT.txt BSD-3-Clause.txt`

Reason is to be able to compare successful/unsuccessful builds and see if the error because of an updated version.